src/state/apis/snapshot/snapshot.ts

```diff
@@ -79,9 +81,15 @@ export const snapshotApi = createApi({
             return bnOrZero(VotingPowerSchema.parse(votingPowerUnvalidated).vp)
           }),
         )
-        const data = BigNumber.sum(...votingPowerResults).toString()
-        console.log('addresses', evmAddresses, 'FOX voting power', data)
-        return { data }
+        const foxHeld = BigNumber.sum(...votingPowerResults).toNumber()
+
+        // Return an error tuple in case of an invalid foxHeld value so we don't cache an errored value
+        if (isNaN(foxHeld)) {
+          const data = `Invalid foxHeld value: ${foxHeld}`
```

Comment:
if `foxHeld` is `NaN` (it can't be anything else since its the output of a sum above), we dont need to include the value in the message
```suggestion
          const data = 'NaN foxHeld value'
```

Reply:
The latter sounds more declarative though won't serialize as a message since it's a `BigNumber[]` so let's go with the former: 367138b84e